crates/cheatcodes/src/env.rs

```diff
@@ -258,7 +258,10 @@ pub fn set_execution_context(context: ForgeContext) {
 }
 
 fn env(key: &str, ty: &DynSolType) -> Result {
-    get_env(key).and_then(|val| string::parse(&val, ty).map_err(map_env_err(key, &val)))
+    get_env(key).and_then(|val| {
+        ensure!(!val.is_empty(), "env var value cannot be empty");
```

Comment:
an empty env var is valid.
I assume some users do this to erase the value, so some would treat "" as delete, but I think we shouldn't do this and force them to properly unset envvars instead?